### ISSUE TITLE
After trying to run Pi-Apps from CLI, LXDE theme reflects Pi-Apps dark theme 

### ISSUE THREAD

**Issue by Crilum** [Botspot/pi-apps#1368]
### Confirmations

- [X] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)
- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.

### What happened?

See title ^

### Description

After I tried to run Pi-Apps from CLI with no Xserver, when I started LXDE, the whole system theme looked like the Pi-Apps Dark Mode, which actually doesn't look bad at all, but it definitely shouldn't of happened:

![2022-01-10-214600_2560x1440_scrot](https://user-images.githubusercontent.com/91354257/148882753-58b8eea8-96f6-46f8-ada7-7fdd18f31fa4.png)

After looking at @cycool29's theme PR, I think this has something to do with setting the `GTK_THEME` variable..

### What is your operating system?

RPiOS 11 (Bullseye)

**Comment by cycool29** [comment]
I can't reproduce that on my side.

Just for confirmation. the command you ran in your terminal is `pi-apps`, right?

**Comment by open-sorcerer64** [comment]
can you please share your neofetch

**Comment by Crilum** [comment]
Yeah, I don't know what happened.. I tried to reproduce it, but couldn't either. I had my configuration set to boot to the CLI, and then I would run `startx` if I needed the Desktop. I switched it back to `Boot to Desktop`, and everything was normal (you know, the default white theme). I actually like the dark theme better, so I booted into CLI, ran pi-apps (it obviously failed), ran `startx`, and the theme was the same. I then looked at the `api` script to see how it set the theme to dark mode, and set `GTK_THEME=Adwaita-dark` manually, started LXDE, and it was back to dark mode. 

I guess what happened was just a fluke... :man_shrugging:
